boards/nordic/nrf54lm20apdk/board.yml

```diff
@@ -4,6 +4,9 @@ board:
   vendor: nordic
   socs:
   - name: nrf54lm20a
+    variants:
+    - name: xip
```

Comment:
should have flash runner config added - see https://github.com/zephyrproject-rtos/zephyr/blob/main/boards/nordic/nrf54l15dk/board.yml#L17